main.go

```diff
@@ -4,7 +4,6 @@ import (
 	"os"
 	"path"
 
-	log "github.com/Sirupsen/logrus"
```

Comment:
You'll need to also import `"github.com/docker/machine/log"` here (`log` is referenced on ln 75)